services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -149,11 +149,11 @@ async def task_pull_user_servcices_docker_images(
 ) -> None:
     assert shared_store.compose_spec  # nosec
 
-    progress.update(message="started pulling user services", percent=0)
+    await progress.update(message="started pulling user services", percent=0)
 
     await docker_compose_pull(app, shared_store.compose_spec)
 
-    progress.update(message="finished pulling user services", percent=1)
+    await progress.update(message="finished pulling user services", percent=1)
 
```

Comment:
here it would be nice to have a context manager... which is available in the progressbar

Reply:
I see what you mean, but the progress is always auto terminated to 100% when the task is completed